convert2rhel/toolopts.py

```diff
@@ -193,7 +194,13 @@ def _register_options(self):
             action="store_true",
             help="Automatically recognize the system as eus, utilizing eus repos."
             " 8.6 systems do not require this option as they are recognized as eus automatically."
-            " This option is meant for 8.8+ systems.",
+            " This option is meant for el8.8+ systems.",
+        )
+        self._shared_options_parser.add_argument(
+            "--els",
+            action="store_true",
+            help="Automatically recognize the system as els, utilizing els repos."
+            " This option is meant for el7 systems.",
```

Comment:
I'm wondering about the use of the word `automatically` here and in the `--eus` option... For some reason, it does not sound as automatic as I would think. 

I mean, whenever I think about `automatically` enabling something, I would expect the tool to do it for me, without me asking for it, like the old behavior. If I'm using either `--els` or `--eus`, I'm explicitly saying that I know that my system is within that supported range for both options and I want it. If convert2rhel will enable/disable or do some magic on top of it, then I don't see it as `automatic` anymore, but rather, user defined... Not sure if that makes sense... 

But again, this is not a blocker, just wanted to open the discussion to see if other thinks the same.

Reply:
Good point, does `Explicitly recognize` sound better in this case?

Reply:
Yes, I think sounds better.